charts/kubefed/charts/controllermanager/crds/crds.yaml

```diff
@@ -1075,9 +1067,9 @@ spec:
     kind: ReplicaSchedulingPreference
     listKind: ReplicaSchedulingPreferenceList
     plural: replicaschedulingpreferences
-    singular: replicaschedulingpreference
     shortNames:
     - rsp
+    singular: replicaschedulingpreference
```

Comment:
This proves that generation is working correctly now. This was manually added incorrectly (bad key ordering) in https://github.com/kubernetes-sigs/kubefed/pull/1279 and because the generated CRDs were never moved to the correct location, validation didn't pick up on this.